mdagent/tools/base_tools/preprocess_tools/pdb_get.py

```diff
@@ -181,7 +182,7 @@ def small_molecule_pdb(self, mol_str: str) -> str:
                 m = Chem.AddHs(m)
             except Exception:
                 pass
-            Chem.AllChem.EmbedMolecule(m)
+            AllChem.EmbedMolecule(m)
```

Comment:
```suggestion
            Chem.AllChem.EmbedMolecule(m)
```

Reply:
if I do this, it'll give an error and fail unit tests, `AttributeError: module 'rdkit.Chem' has no attribute 'AllChem'`. I thought it might be due to version differences, so I created new conda following README instructions and still got the same error. I follow the same import statement according to rdkit docs and it worked. no clue why it worked in other branches.

Reply:
Gotcha, no worries